runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1247,6 +1247,42 @@ gcParseXXArguments(J9JavaVM *vm)
 		}
 	}
 
+	{
+#if defined(J9VM_ARCH_X86)
+		IDATA portableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXPORTABLESHAREDCACHE, NULL);
+		IDATA noportableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXNOPORTABLESHAREDCACHE, NULL);
+		if (portableSharedCacheIndex != noportableSharedCacheIndex) {
+			/* At least one option is set. Find the right most one. */
+			if (portableSharedCacheIndex > noportableSharedCacheIndex) {
+				extensions->portableSharedCacheEnabled = true;
+			} else {
+				extensions->portableSharedCacheEnabled = false;
+			}
+		} else {
+			extensions->shouldForceCeilingShiftForContainers = true;
+		}
+
```

Comment:
we can skip code below if `extensions->shouldForceCeilingShiftForContainers` is not set to `true`